packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/native-select/Controlled.jsx

```diff
@@ -22,8 +22,8 @@ const ControlledExample = () => {
         value={value}
         className={cx('form-select')}
       />
-      <button type="button" onClick={() => setValue()}>Clear</button>
-      <button type="button" onClick={() => setValue('mercedes')}>Set Mercedes</button>
+      <button type="button" className={cx('controlled-button')} onClick={() => setValue(undefined)}>Clear</button>
+      <button type="button" className={cx('controlled-button')} onClick={() => setValue('mercedes')}>Set Mercedes</button>
```

Comment:
I would suggest that we do not use anonymous functions in our examples.  This would be a bad practice that we would not want propagated as it creates a new function for each click.  We would want to use a defined function for the click event.

Reply:
Added named function instead of anonymous function - [e5e0ac0](https://github.com/cerner/terra-core/pull/3860/commits/e5e0ac0037181db80a83493913e9bd04e58e66df)